app/models/maintenance_tasks/run.rb

```diff
@@ -41,6 +41,13 @@ class Run < ApplicationRecord
 
     scope :active, -> { where(status: ACTIVE_STATUSES) }
 
+    # Ensure ActiveStorage is in use before preloading the attachments
+    scope :with_attached_csv, -> do
+      if ActiveStorage::Attachment.table_exists?
+        includes(csv_file_attachment: :blob)
```

Comment:
Why do we have to re-implement it ourselves? Can we rely on `with_attached_csv_file` instead?